syntax = "proto3";

package fdproto;

option cc_generic_services = true;

message ResultCode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

message GetFriendsListRequest
{
    uint32 userid = 1;
}

// 定义登录响应消息类型
message GetFriendsListResponse
{
    ResultCode result = 1;
    repeated bytes friends = 2;
}

service FriendServiceRpc
{
    rpc GetFriendLists(GetFriendsListRequest) returns(GetFriendsListResponse);
}